src/createSession.js

```diff
@@ -1,3 +1,35 @@
+import axios from 'axios';
+
+const libraryPackage = require('../package.json');
+
+const logAnalyticsEvent = (apiKey, sessionId, action, connectionId) => {
+  const body = {
```

Comment:
Although, I think we don't have any mandatory field in the client log, it makes sense to send the guid as well